builder/dockerfile/dispatchers.go

```diff
@@ -71,6 +71,10 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 			return errBlankCommandNames("ENV")
 		}
 
+		if runtime.GOOS == "windows" {
+			// Case insensitive environment variables on Windows
+			args[j] = strings.ToUpper(args[j])
```

Comment:
It means the comment written by the builder (which is used for hash generation to determine if a layer has changed) is consistently uppercase.